cardigan/fractal-app.js

```diff
@@ -68,25 +69,30 @@ fractal.components.set('default.collator', function(markup, item) {
   `);
 });
 
-fractal.docs.set('path', dir('/views/docs'));
+fractal.docs.set('path', serverDir('views/docs'));
 fractal.docs.set('ext', 'njk');
 fractal.docs.engine(nunjucks);
 
 fractal.web.set('static.path', dir('./../dist'));
-fractal.web.set('builder.dest', dir('./../cardigan'));
+fractal.web.set('builder.dest', '.dist');
 
 const cardiganTheme = mandelbrot({
   skin: 'navy',
-  styles: ['default', 'dist-styles/styleguide.css']
+  styles: ['default', '/dist-styles/styleguide.css'],
+  favicon: '/cardigan-theme/assets/favicon.ico'
 });
-cardiganTheme.addLoadPath(`${__dirname}/cardigan-theme`);
-cardiganTheme.addLoadPath(`${__dirname}/views`);
-cardiganTheme.addStatic(path.join(__dirname, '/cardigan-theme'), '/cardigan-theme');
-cardiganTheme.addStatic(path.join(__dirname, '../dist/assets/css/'), '/dist-styles');
+cardiganTheme.addLoadPath(serverDir('cardigan-theme'));
+cardiganTheme.addLoadPath(serverDir('views'));
+cardiganTheme.addStatic(serverDir('cardigan-theme'), '/cardigan-theme');
+cardiganTheme.addStatic(dir('./../dist/assets/css/'), '/dist-styles');
 fractal.web.theme(cardiganTheme);
 
 export default fractal;
 
 function dir(relPath) {
```

Comment:
I know this isn't part of this PR, but since it's referenced in `serverDir` I'll mention it – is `path.resolve` the appropriate method here? I'd have thought that `path.join(__dirname, relPath);` would make more sense?